builds/e2e/e2e.yaml

```diff
@@ -104,13 +210,13 @@ jobs:
     - template: templates/e2e-clear-docker-cached-images.yaml
```

Comment:
Same as above. I don't think we need to clear the docker cached images for 1ES Windows agents.